src/Core/Config/Config_ForClass.php

```diff
@@ -9,7 +9,7 @@ class Config_ForClass
     /**
      * @var string $class
      */
-    protected $class;
+    public $class;
```

Comment:
I wouldn't even understand why it would need to be public for debugging purposes?